tests/org.eclipse.fordiac.ide.test.ui/META-INF/MANIFEST.MF

```diff
@@ -33,4 +35,4 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.swt,
  org.hamcrest.core,
  org.eclipse.draw2d,
- org.hamcrest.library;bundle-version="[1.3.0,2.0.0)"
+ org.hamcrest.library;bundle-version="[1.3.0,2.2.1)"
```

Comment:
I’m a bit confused about the this change, specifically for:
As version 2.2.1 of org.hamcrest.library is not available yet, and the recommended version for SWTBot 4.2.1 is 2.2.0, I wanted to clarify if it is appropriate to proceed with these changes. Should we align with the recommended version 2.2.0 for org.hamcrest.library, or is there another versioning strategy you’d suggest?

Reply:
What you currently see is just a workaround that was found in the SWTBot tests of EGit. I think the solution you proposed is a much cleaner one. Therefore I asked you to create an own PR where only your fix to the MANIFEST.MF file is contained.

Reply:
I have created the new pull request now.